video/cloud-client/analyze/analyze_test.py

```diff
@@ -24,37 +24,37 @@
 BUCKET = os.environ['CLOUD_STORAGE_BUCKET']
 LABELS_FILE_PATH = '/video/cat.mp4'
 FACES_FILE_PATH = '/video/googlework.mp4'
-SAFE_SEARCH_FILE_PATH = '/video/cat.mp4'
+EXPLICIT_CONTENT_FILE_PATH = '/video/cat.mp4'
```

Comment:
We're sure the same video still works for these apparently different analyses?

Reply:
this was a rebranding of save_search_detection --> explicit_content_detection.

Reply:
Oh, I thought the "safe search" analysis was "is this safe?", the inverse of the "explicit content analysis". If it's that confusing, no wonder they rebranded it.

Reply:
yeah indeed it was confusing - safe_search_detection returned things which are not safe, for the user to filter out portions of the video, for example.